XpringKit/Org_Xrpl_Rpc_V1_Transaction+XRPTransaction.swift

```diff
@@ -1,7 +1,13 @@
 import Foundation
 
 internal extension XRPTransaction {
-  init?(transaction: Org_Xrpl_Rpc_V1_Transaction) {
+  private static let javaScriptUtils = JavaScriptUtils()
+
+  init?(getTransactionResponse: Org_Xrpl_Rpc_V1_GetTransactionResponse) {
```

Comment:
I think Keefer once told me that it's more idiomatic in Swift to do something like `init?(from: Org_Xrpl_Rpc_V1_Transaction).  Makes the parameter name more informative and cleans up the call site

Reply:
that may be true, though I wrote this to match the style of the other 8 wrapper classes that Keefer wrote :)  @keefertaylor can let me know if all of them should be updated with a `from` argument label